Dockerfile

```diff
@@ -85,6 +85,10 @@ RUN chmod +x /usr/local/bin/doctor && \
     chmod +x /usr/local/bin/worker-messaging && \
     chmod +x /usr/local/bin/worker-migrations && \
     chmod +x /usr/local/bin/worker-webhooks && \
+    chmod +x /usr/local/bin/worker-stats-usage && \
+    chmod +x /usr/local/bin/worker-stats-usage-dump && \
+    chmod +x /usr/local/bin/stats-resources && \
+    chmod +x /usr/local/bin/worker-stats-resources && \
     chmod +x /usr/local/bin/worker-usage && \
```

Comment:
Let's add a note to remove old ones later, or at least have it written down somewhere.

Reply:
I'll have another PR ready